internal/provider/utils/filtering.go

```diff
@@ -97,3 +101,19 @@ func filterByPossibilities[T string | uint16](value T, possibilities []T) (filte
 	}
 	return true
 }
+
+func filterMultipleByPossibilities[T string | uint16](values []T, possibilities []T) (filtered bool) {
+	if len(possibilities) == 0 || len(values) == 0 {
+		return false
+	}
+
+	pass := true
+	for _, value := range values {
+		if !filterByPossibilities(value, possibilities) {
+			pass = false
+			break
+		}
+	}
+
+	return pass
```

Comment:
Correct me if I'm wrong, but I think this is meant to be:

```suggestion
	for _, value := range values {
		filtered = filterByPossibilities(value, possibilities)
		if filtered {
			return true
		}
	}

	return false
```

Reply:
![image](https://github.com/qdm12/gluetun/assets/18214951/aaf1db87-6a37-4eab-97ca-b80b48d472e5)

It's kinda based on this, but from my understanding, a TRUE evaluation means to filter out the value? Pls read filtering.go since you'll understand it more than me and advise pls

Reply:
You're correct in your understanding of `filterByPossibilities`, but your function returns "do not filter" as soon as any value from `values` is correct; we want to check all values are correct. I'm fixing this 😉